changelog/5883.bugfix.rst

```diff
@@ -0,0 +1 @@
+Updated how HMIMap is selected when using Map from `sunpy.map`.

```

Comment:
```suggestion
`HMIMap` now looks for ``'INSTRUME`` instead of ``'TELESCOP'`` to follow JSOC standards. 
```

Reply:
I object to this suggested change because it makes it sounds like SunPy did something wrong.  We *have* been using a JSOC standard keyword.  As I understand it, the problem is that the Helioviewer JPEG2000 versions of HMI data do not preserve the `TELESCOP` keyword.

Reply:
I will change it.